lms/static/sass/base/_variables.scss

```diff
@@ -6,6 +6,11 @@ $baseline: 20px;
 
 // ====================
 
+// PATH
```

Comment:
I like path configuration being set at the beginning of our variables. Mind syncing this convention up in ``cms/static/sass/_variables.scss``?